### ISSUE TITLE
psi probe not finding logback logs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [psi-probe/psi-probe#310]
```
What steps will reproduce the problem?
1. Create an Ubuntu 10.04 32bit VM
2. Install tomcat 6.0.24
3. Install logback 1.0.1
4. Install a "hello world" app that logs using logback
5. Visit the log tab of probe

What is the expected result?
See all logs, including those generated by logback

What happens instead?
No logback logs show up

What version of Probe are you using?
2.3.1 (but also tested with 2.2.2 - 2.3.0)

What environment (browser version, Tomcat version, JVM version, server OS)?

Browser: Firefox 10.4 ESR
Server OS: Ubuntu 10.04 LTS (32bit, 'server' image)
JVM version: Java 1.6.0_24 (OpenJDK (IcedTea6 1.11.3)
Tomcat: 6.0.24

logback version: 1.0.1
logback.xml is attached.


Please provide any additional information below.
(Attach logs or stack traces as files instead of pasting the contents
here.)

```

Original issue reported on code.google.com by `llowde...@gmail.com` on 14 Aug 2012 at 1:13

Attachments:
* [logback.xml](https://storage.googleapis.com/google-code-attachments/psi-probe/issue-310/comment-0/logback.xml)

**Comment by hazendaz** [comment]
Isn't this resolved now as of 2.4.0?

**Comment by MALfunction84** [comment]
Yes it is. Duplicate of #204.
